### ISSUE TITLE
Request and Object IDs for calibrations including standard stars

### ISSUE THREAD

**Issue by scizen9** [scizen9/sedmpy#29]
Observations of standard stars and calibration lamps cannot be entered into the database without a request id or an object id.  We need entries in the object table for the cal lamps and the standard stars, and we need to figure out how to generate a request id for cal observations as well.

**Comment by amdugas** [comment]
Hi Don,

Are you working on this yet? I have some code on my local machine that should do most of this, but it's from before this repository so I first need to integrate it with the rest of the SedmDb stuff, recent database changes, etc (and does it need to be python3-compliant?). If you aren't already working on this, it is probably best if I handled the header->database part given what I already have done.

If you've already made much progress I can just let you do it so we're not stepping on each others' toes :)

Best,

Alison

On Wed, Oct 3, 2018 at 4:45 PM Don Neill <notifications@github.com<mailto:notifications@github.com>> wrote:

Observations of standard stars and calibration lamps cannot be entered into the database without a request id or an object id. We need entries in the object table for the cal lamps and the standard stars, and we need to figure out how to generate a request id for cal observations as well.

**Comment by scizen9** [comment]
Hi Alison,

The object id problem is basically solved after I found out that there are already lamp and dome objects in the object table.  I think all the standards are in there also.

The remaining issue is the request id.  Richard is working on setting up individual requests for the standard stars observations.  We think the most sensible thing is to have a generic request for the calibration domes and lamps and biases, since an individual request for each of these would result in the request table being dominated by calibration requests.

Right now I have updated the add and update functions for the observation and spec_calib tables.  I may work on the add and update function for the spec table as well, but that is all for the SedmDb.py file.  I am also adding update functions to the AutoReduce.py script for these items, but I  won't be calling them in the pipeline until we finalize the database structure and solve the request id for calibrations issue.

We are moving our weekly SEDM meeting to Mondays at 9am, so that would be a good time to discuss the structure of the database and to assign tasks related to the database.  I hope to see you there!

-Don
